joblib/test/test_parallel.py

```diff
@@ -586,7 +586,9 @@ def test_backend_context_manager():
         assert _active_backend_type() == MultiprocessingBackend
         # check that this possible to switch parallel backends sequentially
         for test_backend in all_backends:
-            yield check_backend_context_manager, test_backend
+            # TODO: parametrize this block later
+            # yield check_backend_context_manager, test_backend
+            check_backend_context_manager(test_backend)
```

Comment:
OK so this is the actual fix for the tests failing with py.test in master. No clue why this yield does not work but the other yield does not seem to cause any issue.